### PR TITLE
Language override

### LINKED ISSUE #24: CustomAttributes not being set
I am trying to override language for intercom, depending on user  settings. 

Here I set language to portugal:
`customAttributes: {"languageOverride": "pt"}
`
but it is not being set in intercom. I can override all the other info (that is not part of customAttributes), but I cant override the language.

Any idea on how to change the language for some users?

### LINKED ISSUE #124: Allow language override
In the JS version of Intercom overriding a language is possible:
```
var intercomSettings = {
  app_id: 'bk7tlzy4',
  ...,
  language_override: 'fr' // Replace with preferred language of user
};
```
But this doesn't seem to be the case for the flutter version.
Any plans on this? :)

### PR DESCRIPTION
**Pull request by deepak786**
Allow overriding the language of the Intercom. 

https://github.com/v3rm0n/intercom_flutter/issues/124
https://github.com/v3rm0n/intercom_flutter/issues/24

### CODE REVIEW

**Comment by deepak786** [comment]
@Zazo032 Check this.
After merging, I will implement this feature for the web https://github.com/v3rm0n/intercom_flutter/pull/111

I have created a separate function to override the language. But it can also be used with updateUser.
Let me know what do you think?

File: lib/intercom_flutter.dart
Comment (Zazo032) [inline lib/intercom_flutter.dart]:
```suggestion
  /// Override the language of Intercom messenger.
```

File: lib/intercom_flutter.dart
Comment (Zazo032) [inline lib/intercom_flutter.dart]: I think this might not be only for Chinese, there are more specific locales that work like this, but I'm not sure how many of those Intercom supports. A link to supported languages can be very useful here

**Review by Zazo032 — COMMENTED** [review]
Thanks @deepak786 for the PR. Since the native call is done through `updateUser`, I think it would be better to do it the same way in the Dart side, since it can be a better UX for devs (docs for `updateUser` can mention all the params it can handle separately.

File: lib/intercom_flutter.dart
Comment (deepak786) [reply lib/intercom_flutter.dart]: I have added the Intercom link.

File: android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt
Comment (Zazo032) [inline android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt]:
```suggestion
    if (language != null) {
```

File: lib/intercom_flutter.dart
Comment (Zazo032) [inline lib/intercom_flutter.dart]:
```suggestion
  /// Updates the attributes of the current Intercom user.
  ///
  /// The [language] param should be an an ISO 639-1 two-letter code such as `en` for English or `fr` for French.
  /// You’ll need to use a four-letter code for Chinese like `zh-CN`.
  /// check this link https://www.intercom.com/help/en/articles/180-localize-intercom-to-work-with-multiple-languages.
  ///
  /// See also:
  ///  * [Localize Intercom to work with multiple languages](https://www.intercom.com/help/en/articles/180-localize-intercom-to-work-with-multiple-languages)
```

**Review by Zazo032 — APPROVED** [review]
Some minor nits regarding docs and style. Also, could you bump package version to `3.1.0` and add this change to `CHANGELOG.md`? I'll merge the PR after that's done

**Comment by Zazo032** [comment]
Thanks!

**Pull request merged**
